contextualized/analysis/effects.py

```diff
@@ -278,6 +279,16 @@ def plot_homogeneous_context_effects(
     Returns:
         None
     """
+
+    # if normalization is implemented, we hope to inverse_transform and show original context axis
+    if inverse_transform and getattr(model, "normalize", False):
+        if hasattr(model, "scaler_C") and model.scaler_C is not None:
+            # C = model.scaler_C.inverse_transform(C)
+            # C = inverse_transform_preserve_df(model.scaler_C, C)
```

Comment:
Various debugging comments like this still included. Only include explanatory comments, not comments intended for testing or switching behavior. Please remove and run black formatting. See CONTRIBUTING.md.